pyproject.toml

```diff
@@ -82,6 +82,7 @@ docs = [
     "sphinx-notfound-page~=0.5",
     "sphinxext-rediraffe~=0.2.4",
     "sphinx-sqlalchemy~=0.1.1",
+    "sphinx-intl[transifex]~=2.0.1",
```

Comment:
What does the `transifex` extra install and is it required? Because in the `requirements` files there is just the `sphinx-intl` added.

Reply:
I am not sure whether the extra requirement of should be included in the `requirements-*.txt` file, I refer to the `sqlalchemy[mypy]` in `pre-commit` which not in the requirements-*.txt list.

Reply:
Yeah, regardless, I just realized that those requirements file are mostly to record a full working environment for when we need to go back into the past and install an older version. But there we won't need the transifex dependencies. They are only relevant for this workflow and there they are installed through the abstract dependencies from the `pyproject.toml`. So I think this is all good.